### DISCUSSION TITLE
Math Expansion Explanation

### DISCUSSION THREAD

**Discussion by DeveloprOfThings** [RayTracing/raytracing.github.io#938] [Q&A] [answered]
I'm having trouble understanding how you expanded the following equation:`(A+tb−C)⋅(A+tb−C)=r2`
to the final form:
`t2b⋅b+2tb⋅(A−C)+(A−C)⋅(A−C)−r2=0`
 When trying myself, I ended up with the following equation:
`A^2-2AC+t^2b^2+2Atb-2tbC+C^2` 
which is significantly different from what you derived. Could you provide any insight into where I went wrong with the equation above?

**Comment by trevordblack** [comment, marked as answer]
`(A+tb−C)⋅(A+tb−C)=r_2`
`(tb + (A-C))⋅(tb + (A-C)) = r_2`
`(tb)⋅(tb) + 2*tb⋅(A-C) + (A-C)⋅(A-C) = r_2`
`t^2b⋅b+ 2*tb⋅(A-C) + (A-C)⋅(A-C) - r_2 = 0`

**Reply by DeveloprOfThings** [reply]
Thank you for such a quick response. The explanation was exactly what I was missing.
